VERSION

```diff
@@ -1 +1 @@
-0.1.0
\ No newline at end of file
+0.1.1
```

Comment:
For this a minor bump is more appropriate than patch (even though we are in 0.x).
```suggestion
0.2.0
```